src/main/java/de/bonndan/nivio/input/Seed.java

```diff
@@ -67,12 +68,13 @@ private URL asURL(String s) {
         if (!StringUtils.hasLength(demo)) {
             return demoFiles;
         }
+        String value = ConfigurableEnvVars.DEMO.value().orElse("");
 
         Path currentRelativePath = Paths.get("");
         String absPath = currentRelativePath.toAbsolutePath().toString();
         try {
-            demoFiles.add(new File(absPath + "/src/test/resources/example/pet_clinic.yml").toURI().toURL());
-            if (demo.equalsIgnoreCase("all")) {
+            demoFiles.add(new File(absPath + "/src/test/resources/example/example_k8s.yml").toURI().toURL());
```

Comment:
Nope 😄 

We should introduce that the demo file can be chosen by name instead of "all" or "1"